core-services/prow/02_config/opendatahub-io/opendatahub-operator/_prowconfig.yaml

```diff
@@ -2,7 +2,10 @@ tide:
   merge_method:
     opendatahub-io/opendatahub-operator: squash
   queries:
-  - labels:
+  - includedBranches:
+    - feature/rearchitecture
```

Comment:
wonder if here uses "feature/rearchitecture" then then file name need to match it. 
so if we have a branch "feature-rearchitecture" which should be fine.